test/cli/test/inspect.test.ts

```diff
@@ -54,7 +54,7 @@ test('--inspect-brk stops at test file', async () => {
 })
 
 async function createChannel(url: string) {
-  const ws = new WebSocket(url)
+  const ws = new WebSocket(url, { allowSynchronousEvents: false })
```

Comment:
This test times out on linux due to the breaking change in https://github.com/websockets/ws/releases/tag/8.17.0 https://github.com/websockets/ws/pull/2221.

I can reproduce it locally too. For now, I flipped it back, so it works same.